src/adhocracy_meinberlin/adhocracy_meinberlin/workflows/bplan.yaml

```diff
@@ -3,15 +3,15 @@ states:
   participate:
     acm:
     acm:
-      principals:                  [anonymous, participant, moderator, creator, initiator, admin]
+      principals:                  [everyone, participant, moderator, creator, initiator, admin]
       permissions:
-        - [create,                  A,         ~,           A,          A,      A,         ~]
-        - [create_proposal,         A,         ~,           ~,          ~,      ~,         ~]
-        - [edit_proposal,           ~,         ~,           ~,          ~,      ~,         ~]
+        - [create,                  A,        ~,           A,          A,      A,         ~]
+        - [create_proposal,         A,        ~,           ~,          ~,      ~,         ~]
+        - [edit_proposal,           ~,        ~,           ~,          ~,      ~,         ~]
   evaluate:
```

Comment:
If we allow Everyone to create proposal, we also allow it to authenticted user. 
But for bplan we don't want this, or?